bundles/binding/org.openhab.binding.zwave/database/swiid/SW-ZCS-1.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+    <Model>SwiidInter</Model>
+    <Label lang="en">Z-Wave Cord Switch</Label>
+    <CommandClasses>
+    	<Class><id>0x20</id></Class>
+    	<Class><id>0x25</id></Class>
+    	<Class><id>0x27</id></Class>
+    	<Class><id>0x72</id></Class>
+    	<Class><id>0x85</id></Class>
+    	<Class><id>0x86</id></Class>
+    </CommandClasses>
+    <Configuration>
+    	<Parameter>
+    		<Index>1</Index>
+    		<Type>list</Type>
+    		<Default>00</Default>
+    		<Size>1</Size>
+			<Label lang="en">Selects the switch mode. The value 255 is normal mode (switch on and off) and default operation.</Label>    			<Item>
```

Comment:
Should this label be the <help> tag for this parameter (ie at line 31 below)? I don't think it should be here anyway.